app/models/payment.rb

```diff
@@ -1,9 +1,17 @@
 class Payment < ApplicationRecord
-  belongs_to :claim
+  has_many :claims
```

Comment:
Did you consider using a `has_one` association instead of this and the temporary `claim` method?

Reply:
Nope, it didn't occur to us. But I think it's a good idea, in the spirit of small changes. I've changed it.